recover-id-object/scripts/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG build_image=rust:1.70.0
+ARG base_image=debian:buster-slim
+FROM ${build_image} AS build
+
+WORKDIR /usr/app/recover-id-object
+
+COPY recover-id-object/Cargo.toml recover-id-object/Cargo.lock .
```

Comment:
This improves the build time drastically given that you only have source code changes. I do not see that being needlessly.

The dependencies should always be defined in the Cargo files.

Reply:
I do believe there is reason for this as it does increase the performance.

But since this is a simple and small codebase, which does not have a long build time, increasing the build simplicity over the little performance this gives me does seem like a fair enough point.

Also it ensures that we do not diverge from the rest for little reasons.